Formula/uniutils.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
```

Comment:
I think that was added by emacs automatically when I inserted the ü into the file. Editor encoding hints are something that should definitely be taken out?